cmd/pets/down.go

```diff
@@ -0,0 +1,40 @@
+package pets
+
+import (
+	"fmt"
+	"os"
+	"syscall"
+
+	"github.com/spf13/cobra"
+	"github.com/windmilleng/pets/internal/proc"
+)
+
+var DownCmd = &cobra.Command{
+	Use: "down",
+	Run: func(cms *cobra.Command, args []string) {
+		procfs, err := proc.NewProcFS()
+		if err != nil {
+			fmt.Println(err)
+			os.Exit(1)
+		}
+
+		procs, err := procfs.ProcsFromFS()
+		if err != nil {
+			fmt.Println(err)
+			os.Exit(1)
+		}
+
+		if len(procs) == 0 {
+			fmt.Println("No pets running")
+			return
+		}
+
+		fmt.Printf("Process ID\tName\n")
+		for _, p := range procs {
+			err := syscall.Kill(p.Pid, syscall.SIGINT)
```

Comment:
SIGINT is the "please shut down" signal...

in the future, it might be good to wait for the pets to go down? what should happen if they don't exit politely? should we report it to the user or kill them?

probably makes sense to punt on these questions for now but maybe leave a comment?